src/ape_ethereum/ecosystem.py

```diff
@@ -382,6 +384,9 @@ class Ethereum(EcosystemAPI):
 
     fee_token_symbol: str = "ETH"
 
+    # NOTE: This gets appended to Ape's root User-Agent string.
+    request_header: dict = {"User-Agent": "ape-ethereum"}
```

Comment:
hmm, wonder if could do some magic with a field validator to sneak out the module that was subclassing this module and insert it's name, something like `cls.__package__.__name__` so that the right package name appears (`ape-ethereum`, `ape-polygon`, etc.)

Might be too fancy though

Reply:
I think that works nicely for these plugin-level ones, I will give it a shot.
(Note: it does not work as great for the config-level ones, but not going to get into that).

Reply:
done!